examples/webgl_interactive_cubes_gpu.html

```diff
@@ -136,7 +136,7 @@
 
 					// give the geometry's vertices a color corresponding to the "id"
 
-					applyVertexColors( geometry, color.setHex( i ) );
+					applyVertexColors( geometry, color.setHex( i, THREE.NoColorSpace ) );
```

Comment:
nit: I would suggest LinearSRGBColorSpace, no functional difference but it identifies that we're encoding IDs as colors, rather than saying this isn't a color.

Reply:
>it identifies that we're encoding IDs as colors, rather than saying this isn't a color.

I would argue it's not a color; it's an ID.

Reply:
No objection 👍